api/src/business_ar_api/models/filing.py

```diff
@@ -131,6 +132,46 @@ def find_business_filings_by_status(
         )
         return query.all()
 
+    @classmethod
+    def get_next_ar_fiscal_year(cls, business_id: str) -> int | None:
+        try:
+            status = [Filing.Status.COMPLETED, Filing.Status.PAID]
+            query = (
+                db.session.query(Filing)
+                .filter(Filing.business_id == int(business_id))
+                .filter(Filing.status.in_(status))
+                .order_by(Filing.fiscal_year.desc())
+            )
+            filings = query.all()
```

Comment:
Completed* filings in GCP this method fetches.

So when a filing is first made the status goes to PAID, and then when it is synced with Colin (process-paid-filings) it goes to COMPLETED.

 This is meant to check for the latest synced or unsynced filing in GCP. The reason is that when fetching the next filing to complete it can also check GCP incase there is a filing that has been completed that has not yet been synced with Colin.